source/_docs/ecosystem/ios.markdown

```diff
@@ -31,7 +31,10 @@ The `ios` component is the companion component for the Home Assistant iOS app. W
 
 Loading the `ios` component will also load the [`device_tracker`](/components/device_tracker), [`zeroconf`](/components/zeroconf) and [`notify`](/components/notify) platforms.
 
-In order to save credentials and adjust settings within the iOS Home Assistant app, the optional [`api_password`](/components/http/#api_password) must be set. 
+Home Assistant for iOS app supports the new authentication system introduced in Home Assistant 0.77.
+
+In order to save credentials and adjust settings within the Home Assistant for iOS app prior to version 1.5.0, the optional [`api_password`](/components/http/#api_password) must be set. Using legacy authentication is not recommended.
```

Comment:
The app also supports `legacy API password` (https://www.home-assistant.io/docs/authentication/providers/#legacy-api-password), if you are still using it. Using legacy authentication is not recommended.

Reply:
Yeah we should not mention it in the docs in the intro, as people might think they need to include it.

Reply:
Removed the part as requested by balloob.